docs/topics_5/web-analyze-apps.adoc

```diff
@@ -3,17 +3,17 @@
 [id='using_web_console_analyze_apps_{context}']
 = Analyzing applications with the {WebName}
 
-In order to use the {WebName} to analyze applications, you must create a project. Each project is a way to group applications for analysis and specifies the settings to use during the analysis of its applications. The analysis process generates reports that provide information about the changes necessary for a migration or modernization effort.
+To use the {WebName} to analyze applications, you must create one or more projects. Each project is a way to group applications for analysis and specifies the settings to use during the analysis of its applications. The analysis process generates reports that provide information about the changes necessary for a migration or modernization effort.
 
 Follow these steps to use the {WebName} to analyze your applications and review the results.
 
-. Add a project.
+. Create a project.
 . Add applications to the project.
-. Specify the analysis configuration.
+. Configure the analysis.
 . Execute the analysis.
 . View the results of the analysis.
 
 [NOTE]
 ====
-Click the *Projects* link at the top of the {WebName} to return to the project list at any time. Visit the help menu for an *About* page and a link to the {ProductShortName} documentation.
+Click *Projects* at the top of the {WebName} navigation bar to return to the project list at any time. Visit the help menu for an *About* page that includes a link to the {ProductShortName} documentation.
 ====
```

Comment:
This information seems out of place. I would remove the note here and move the line about returning to the project list in a module where you are not in the project list. Maybe in configuring analysis. The info does not make sense here.

Remove the "Visit the help menu" line. That makes no sense because if they're reading about it here, they already know where to find the docs. If they see the help menu in the UI, they don't need to read it here.